web/src/components/newsContent.js

```diff
@@ -1,33 +1,24 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import BlockText from './block-content/blockText';
 
-export default function NewsContent() {
+export default function NewsContent({ data }) {
+    const { _rawNewsBody, newsSubTitle, newsTitle } = data;
     return (
         <section className="container my-24 xl:w-1/2">
             <h2 className="text-3xl font-bold text-brand-1 my-1">
-                Service/Plane
+                {newsTitle}
             </h2>
             <h3 className="font-light leadership-accent mb-12 text-brand-4">
-                Detail
+                {newsSubTitle}
             </h3>
-            <p className="leading-relaxed">
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-                Vestibulum auctor erat pretium ligula dignissim cursus eleifend
-                quis sapien. Pellentesque pellentesque aliquet tortor ut
-                hendrerit. Etiam imperdiet orci rutrum consequat cursus. Fusce
-                tincidunt lobortis mauris, a condimentum sapien rutrum ac.
-                Phasellus tempor augue nec enim hendrerit, id pretium quam
-                efficitur. Aenean nec malesuada dolor, vitae gravida dui. Donec
-                non vulputate diam. Nulla fermentum massa sit amet suscipit
-                posuere. Maecenas in lorem commodo, varius neque vitae, vehicula
-                dolor. Sed ut viverra eros. Orci varius natoque penatibus et
-                magnis dis parturient montes, nascetur ridiculus mus. Nam
-                convallis arcu a quam tristique placerat. Maecenas egestas, nibh
-                et ultricies tristique, purus magna sodales nisi, sit amet
-                efficitur justo massa accumsan augue. Proin condimentum dolor
-                elementum augue fermentum pretium. Phasellus consequat diam vel
-                ante placerat fermentum sit amet vitae velit. Quisque vel ante
-                ac erat accumsan tincidunt.
+            <p className="leading-relaxed prose">
```

Comment:
I think this should be a `div`. The `<BlockText>` component renders `<div>`'s and they can't be descendants of `<p>`'s